core/jazz_acl/index.js

```diff
@@ -69,6 +73,12 @@ async function processACLRequest(event, config) {
       }
 
       await exportable.processScmPermissions(config, serviceId, policies, 'add');
+
+      logger.info("vault is enabled. global.globalConfig.VAULT.IS_ENALED : " + global.globalConfig.VAULT.IS_ENABLED)
```

Comment:
nitpick: typo 'IS_ENABLED`

Reply:
Removed the log since we don't want the log over here